resources/game-design/stages/test-stage.tmx

```diff
@@ -0,0 +1,240 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<map version="1.4" tiledversion="1.4.3" orientation="orthogonal" renderorder="right-down" width="20" height="100" tilewidth="16" tileheight="16" infinite="0" nextlayerid="5" nextobjectid="30">
+ <tileset firstgid="1" source="../tilesets/space-tileset.tsx"/>
+ <layer id="2" name="details" width="20" height="100">
+  <data encoding="csv">
+0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,
```

Comment:
is this the best way to represent? I feel it will be a lot of zeroes. how about a list of entities, with x, y, and type?

Reply:
This is how tiled works, my idea on using tiled is so that we can use tiled on our game, and maybe notice things to improve.

This specific layer you are looking will probably just be parsed by the tiled.dart and we will not need to worry much about it